src/index.js

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-unused-vars */
+
```

Comment:
- You have done great so far. :+1: :+1: Saving and fetching the scores from an API works great. But there are some changes you need to make before moving to the next milestone. :+1: :+1: 

- [ ] Following the best practices, you are not meant to use any linters diable rule in your codebase. Linters are there to make sure your code follows the best standard, I'll suggest you remove this and fix the error. :+1: